__tests__/api.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require("../app")
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => {
+    return seed(testData)
+});
+afterAll(() => {
+    return db.end()
+});
+
+describe('GET /api/topics', () => {
+    test('200: responds with an array of topic objects', () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                body.data.forEach((topic) => {
+                    expect(topic).toMatchObject({
+                        description: expect.any(String), slug: expect.any(String)
+                    })
+                })
+            })
+    })
+    test('404: responds with an appropriate message when an incorrect url is used', () => {
+        return request(app)
+            .get('/api/topicz')
+            .expect(404)
```

Comment:
At the moment, this 404 is being thrown by the default handler built into express. If you want to keep this test, then please add your own custom 404 error handler